js/main.js

```diff
@@ -1,124 +1,344 @@
+    .style
+    .boxShadow = '#b3923a 1.5px 1.5px 1.5px 1.5px';
+}
+//
 
 //функция которая делает заданый элемент невидимым
 function visibility(nameID) {
-    document.getElementById(`${nameID}`).style.visibility = "hidden";
+  document.getElementById(`${nameID}`).style.visibility = 'hidden';
 }
-//////////////////////////////////////////////////
+//
+
+//Функция события нажатия Enter
+function pressButtonEnter(event) {
+  if (event.keyCode === 13) {
+    start();
+  }
+}
+document.addEventListener('keydown', pressButtonEnter);
+//
+
+//функция которая отрисовует игровые очки
+function drawScore(score) {
+//рисуем прямоугольник
+  ctx.beginPath();
+  ctx.moveTo(0, 0);
+  ctx.lineTo(0, 35);
+  ctx.lineTo(canvas.width, 35);
+  ctx.lineTo(canvas.width, 0);
+  ctx.lineTo(0, 0);
+  ctx.strokeStyle = '#142743';
+  ctx.stroke();
+  ctx.fillStyle = '#142743';
+  ctx.fill();
+  ctx.closePath();
+  //рисуем арбуз
+  ctx.drawImage(scoreImg, 5, 5, 25, 25);
+  //записываем очки рядом
+  ctx.fillStyle = 'white';
+  ctx.font = '20px Arial';
+  ctx.fillText(score, 40, 25);
+}
+//
+
+//Функция которая запускаеться при нажатие на кпопку 'Press Enter to Start Game'
 function start() {
-    //Отключаем кнопку
-    visibility('start_game');
-    document.getElementById('start_game').setAttribute('disabled', 'disabled');
-    //запускаем перед началом игры анимированый таймер
-    let count = 0;
-   for (let i = 3 ; i >= 1 ; i-- ) {
-    setTimeout( ()=>{ document.getElementById(`text${i}`).style.animationPlayState='running'; }, count*1000 );
+//Убираем событие старта игры
+  document.removeEventListener('keydown', pressButtonEnter);
+  //Отключаем кнопку
+  visibility('start_game');
+  document.getElementById('start_game').setAttribute('disabled', 'disabled');
+
+  //Добавляем рестарт
+  document.getElementById('difficult').style.height = '250px';
+  document.getElementById('restart').style.visibility = 'visible';
+
+  //Отключаем смену сложности
+  document.getElementById('changer1').setAttribute('disabled', 'disabled');
+  document.getElementById('changer2').setAttribute('disabled', 'disabled');
+  document.getElementById('changer3').setAttribute('disabled', 'disabled');
+
+  //запускаем перед началом игры анимированый таймер
+  let count = 0;
+  for (let i = 3; i >= 1; i--) {
+    setTimeout(() => {
+      document.getElementById(`text${i}`).style.animationPlayState = 'running';
+      sound_of_horn1.play();
+    }, count * 1000);
     count++;
-   }
-///////////////////////////////////////////////////
+  }
 
-//Основная функция, которая отрисовует игру
-function draw() {
-    ctx.drawImage(background, 0, 0);
-    ctx.drawImage(bowl, refr.x, refr.y, 70, 70);
+  //Основная функция, которая отрисовует игру!!!!
+  function draw() {
+    //Включаем игру когда не стоит пауза
+    if (pause === false) {
+    //Проверяем какую музыку запустить
+      if (speedCount === 15) {
+        easy_music.play();
+      } else if (speedCount === 10) {
```

Comment:
15 and 10 - numbers to named constants
![matrix-numbers-gif-6](https://user-images.githubusercontent.com/4405297/84580469-023c3080-ade0-11ea-8ef5-41cb8785a709.gif)